pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -56,18 +56,18 @@ in stdenv.mkDerivation (finalAttrs: rec {
       fetchurl
         {
           url = "https://isabelle.in.tum.de/website-${dirname}/dist/${dirname}_macos.tar.gz";
-          sha256 = "0b84rx9b7b5y8m1sg7xdp17j6yngd2dkx6v5bkd8h7ly102lai18";
+          sha256 = "1x6pcngibmxsn27x1m6nmrm9gfz2ysg0j0l7jgzqsd675bcrcm6i";
         }
     else if stdenv.hostPlatform.isx86
     then
       fetchurl {
         url = "https://isabelle.in.tum.de/website-${dirname}/dist/${dirname}_linux.tar.gz";
-        sha256 = "1ih4gykkp1an43qdgc5xzyvf30fhs0dah3y0a5ksbmvmjsfnxyp7";
+        sha256 = "sha256-Go4ZCsDz5gJ7uWG5VLrNJOddMPX18G99FAadpX53Rqg=";
       }
     else
       fetchurl {
         url = "https://isabelle.in.tum.de/website-${dirname}/dist/${dirname}_linux_arm.tar.gz";
-        hash = "sha256-qI/BR/KZwLjnkO5q/yYeW4lN4xyUe78VOM2INC/Z/io=";
+        hash = "09s4cwygz465pmyqlf799a0mbbmlzhqj75xnk7raphqa92rp2g2g";
```

Comment:
How did you generate this?

Reply:
```
nix-prefetch-url https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_linux_arm.tar.gz
path is '/nix/store/5d85d8dhss9mja44kw7g8lrv16k8gsqh-Isabelle2023_linux_arm.tar.gz'
09s4cwygz465pmyqlf799a0mbbmlzhqj75xnk7raphqa92rp2g2g
```

same for 
```
nix-prefetch-url https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_macos.tar.gz
path is '/nix/store/my2ccy7ydcjhwi8d1avcfiz6hd5a185s-Isabelle2023_macos.tar.gz'
1x6pcngibmxsn27x1m6nmrm9gfz2ysg0j0l7jgzqsd675bcrcm6i
```

Reply:
It should be generated in SRI format.
How did you build this?
Have you read the OfBorg error messages?

Reply:
@AndersonTorres moved to SRI format, building is done via: `nix build .#isabelle`